deps/chakrashim/include/v8-debug.h

```diff
@@ -26,11 +26,46 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #pragma once
+
 #include <v8.h>
+#include "uv.h"
+#include <queue>
```

Comment:
That's from my memory...

---
In reply to: [82239941](https://github.com/nodejs/node-chakracore/pull/126#discussion_r82239941) [](ancestors = 82239941)